lib/client.js

```diff
@@ -146,10 +146,11 @@ Client.prototype.getWithResponse = function (url, opts, cb) {
 Client.prototype._requestWithRetries = function (method, url, opts, cb) {
   var client = this;
 
+  var timeout = _.isUndefined(opts.retryTimeout) ? this.retryTimeout : opts.retryTimeout;
   var operation = retry.operation({
-    retries: this.retries,
-    minTimeout: this.retryTimeout,
-    maxTimeout: this.retryTimeout
+    retries: _.isUndefined(opts.retries) ? this.retries : opts.retries,
```

Comment:
Being super picky now, but would be good to put this into a variable too `retries`.